xdsl/dialects/riscv.py

```diff
@@ -1704,7 +1704,9 @@ class AddOp(RdRsRsOperation[IntRegisterType, IntRegisterType, IntRegisterType]):
     Adds the registers rs1 and rs2 and stores the result in rd.
     Arithmetic overflow is ignored and the result is simply the low XLEN bits of the result.
 
+    ```
```

Comment:
Is this code?

Reply:
If it is should it also be C code as below?

Reply:
The code below is C pseudocode, it's not currently syntax coloured, but at least the syntax colouring there would be correct. Here there's not really a close in syntax thing that would make sense IMO